package.json

```diff
@@ -98,29 +98,29 @@
     "popper.js": "^1.16.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.9",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/cli": "^7.17.10",
+    "@babel/core": "^7.18.0",
+    "@babel/preset-env": "^7.18.0",
     "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^21.1.0",
-    "@rollup/plugin-node-resolve": "^13.2.1",
-    "autoprefixer": "^10.4.4",
+    "@rollup/plugin-commonjs": "^22.0.0",
+    "@rollup/plugin-node-resolve": "^13.3.0",
+    "autoprefixer": "^10.4.7",
     "babel-plugin-istanbul": "^6.1.1",
     "bundlewatch": "^0.3.3",
     "clean-css-cli": "^5.6.0",
     "cross-env": "^7.0.3",
-    "eslint": "^8.13.0",
-    "eslint-config-xo": "^0.40.0",
+    "eslint": "^8.16.0",
+    "eslint-config-xo": "^0.41.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-qunit": "^7.2.0",
     "eslint-plugin-unicorn": "^42.0.0",
     "find-unused-sass-variables": "^4.0.4",
     "globby": "^11.1.0",
     "hammer-simulator": "0.0.1",
-    "hugo-bin": "^0.85.3",
-    "ip": "^1.1.5",
+    "hugo-bin": "^0.87.1",
+    "ip": "^1.1.8",
```

Comment:
In v5 there was a bump of `ip` from 1.1.5 to 2.0.0. It's not possible here?

Reply:
It is, it's just the author(s) of ip messed up the npm tags and the latest tag is pointing to v1.1.8 since it was released next.

I'll manually update it to v2.0.0.